core/fault/stacktrace/stacktrace_test.go

```diff
@@ -82,7 +82,7 @@ var (
 			{
 				"⇒ core/fault/stacktrace/stacktrace_test.go@36:nested1",
 				"⇒ core/fault/stacktrace/stacktrace_test.go@34:nested3",
-				"⇒ core/fault/stacktrace/stacktrace_test.go@34:nested3",
+				"⇒ core/fault/stacktrace/stacktrace_test.go@36:nested1",
```

Comment:
This seems dubious?

Reply:
Done. Figured out how to make it stable.